src/index.ts

```diff
@@ -37,9 +37,7 @@ async function readCSV(csvFileName = "", batchSize: number = 0) {
     });
     const linesArray = data.split(/\r|\n/).filter((line) => line);
     const columnNames = linesArray?.shift()?.split(",") || [];
-    let beginSQLInsert = `INSERT INTO ${fileAndTableName} (`;
-    columnNames.forEach((name) => (beginSQLInsert += `${name}, `));
```

Comment:
You should check again whether `columnNames` exists or not.
after that, you can perform your `forEach` loop.